website/src/components/sidebar/sidebar.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
Think it might be better to put this in the canvas directory rather than make a new directory because it is still associated with the "Canvas" web page of the app.